src/patternfly/_variables.scss

```diff
@@ -76,6 +76,9 @@
   // Gutter
   --pf-global--gutter: #{$pf-global--gutter};
 
+  // Ratio
+  --pf-global--GoldenRatio: #{$pf-global--GoldenRatio};
```

Comment:
change `GoldenRadio` to `golden-ratio`. Only CSS properties should be in title case.